src/DetailsView/reports/components/report-sections/summary-details.tsx

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { NamedSFC } from '../../../../common/react/named-sfc';
+
+export type SummaryDetailsProps = {
+    summaryProps?: Object;
```

Comment:
I can see why we would want to keep this loose but do you think until we are ready to expand on this component, we can keep strict type checking on this? Thoughts?